packages/pure-finance-lib/src/merkle-claims.js

```diff
@@ -9,6 +9,7 @@ const fetch = require('node-fetch')
 const parseCookieString = require('../lib/parse-cookie')
 const tryParseEvmError = require('../lib/parse-evm-error')
 
+const DEFAULT_MERKLE_ADDRESS = '0xe67516417a934b27cf0c14868f8165b1bc94bf73' // Chain ID 1
```

Comment:
Same comment here re: `Chain ID 1`

Reply:
Removed the default here: https://github.com/hemilabs/pure.finance/commit/50613c94ab27cb3cc111f820f6ba587e14cf0d97